src/utilities.ts

```diff
@@ -132,13 +133,16 @@ export const checkTransactionNotBroadcast = async (
     if (result) throw new Error('Transaction was broadcast!');
   });
 
+const instructionsToFilter = ['ComputeBudget111111111111111111111111111111'];
```

Comment:
Can we add a description to what this is and why we need to do this?

Reply:
added